Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -1990,9 +1990,9 @@ theorem cauchySeq_prod_of_eventually_eq {u v : ℕ → E} {N : ℕ} (huv : ∀ n
   suffices ∀ n ≥ N, d n = d N by exact (tendsto_atTop_of_eventually_const this).cauchySeq.mul hv
   intro n hn
   dsimp
-  rw [eventually_constant_prod _ hn]
+  rw [eventually_constant_prod _ (Nat.add_le_add_right hn 1)]
   intro m hm
-  simp [huv m hm]
+  simp [huv m (Nat.le_of_lt hm)]
```

Comment:
```suggestion
  simp [huv m (le_of_lt hm)]
```

Reply:
```suggestion
  simp [huv m hm.le]
```

Reply:
Eric, I'm not sure if you tried this suggestion, but it doesn't work for me.

Reply:
I did not try it. Weird.